ash/src/vk/native.rs

```diff
@@ -467,139 +459,41 @@ pub struct StdVideoH264HrdParameters {
-        3usize,
-        concat!(
-            "Offset of field: ",
-            stringify!(StdVideoH264HrdParameters),
-            "::",
-            stringify!(reserved1)
-        )
-    );
-    assert_eq!(
-        unsafe { ::core::ptr::addr_of!((*ptr).bit_rate_value_minus1) as usize - ptr as usize },
-        4usize,
-        concat!(
-            "Offset of field: ",
-            stringify!(StdVideoH264HrdParameters),
-            "::",
-            stringify!(bit_rate_value_minus1)
-        )
-    );
-    assert_eq!(
-        unsafe { ::core::ptr::addr_of!((*ptr).cpb_size_value_minus1) as usize - ptr as usize },
-        132usize,
-        concat!(
-            "Offset of field: ",
-            stringify!(StdVideoH264HrdParameters),
-            "::",
-            stringify!(cpb_size_value_minus1)
-        )
-    );
-    assert_eq!(
-        unsafe { ::core::ptr::addr_of!((*ptr).cbr_flag) as usize - ptr as usize },
-        260usize,
-        concat!(
-            "Offset of field: ",
-            stringify!(StdVideoH264HrdParameters),
-            "::",
-            stringify!(cbr_flag)
-        )
-    );
-    assert_eq!(
-        unsafe {
-            ::core::ptr::addr_of!((*ptr).initial_cpb_removal_delay_length_minus1) as usize
-                - ptr as usize
-        },
-        292usize,
-        concat!(
-            "Offset of field: ",
-            stringify!(StdVideoH264HrdParameters),
-            "::",
-            stringify!(initial_cpb_removal_delay_length_minus1)
-        )
-    );
-    assert_eq!(
-        unsafe {
-            ::core::ptr::addr_of!((*ptr).cpb_removal_delay_length_minus1) as usize - ptr as usize
-        },
-        296usize,
-        concat!(
-            "Offset of field: ",
-            stringify!(StdVideoH264HrdParameters),
-            "::",
-            stringify!(cpb_removal_delay_length_minus1)
-        )
-    );
-    assert_eq!(
-        unsafe {
-            ::core::ptr::addr_of!((*ptr).dpb_output_delay_length_minus1) as usize - ptr as usize
-        },
-        300usize,
-        concat!(
-            "Offset of field: ",
-            stringify!(StdVideoH264HrdParameters),
-            "::",
-            stringify!(dpb_output_delay_length_minus1)
-        )
-    );
-    assert_eq!(
-        unsafe { ::core::ptr::addr_of!((*ptr).time_offset_length) as usize - ptr as usize },
-        304usize,
-        concat!(
-            "Offset of field: ",
-            stringify!(StdVideoH264HrdParameters),
-            "::",
-            stringify!(time_offset_length)
-        )
-    );
-}
+const _: () = {
+    ["Size of StdVideoH264HrdParameters"]
+        [::core::mem::size_of::<StdVideoH264HrdParameters>() - 308usize];
+    ["Alignment of StdVideoH264HrdParameters"]
+        [::core::mem::align_of::<StdVideoH264HrdParameters>() - 4usize];
+    ["Offset of field: StdVideoH264HrdParameters::cpb_cnt_minus1"]
+        [::core::mem::offset_of!(StdVideoH264HrdParameters, cpb_cnt_minus1) - 0usize];
```

Comment:
All these changes are going to bump our MSRV to 1.77: https://doc.rust-lang.org/std/mem/macro.offset_of.html